src/assets/family_history_icon.tsx

```diff
@@ -0,0 +1,27 @@
+import { useMantineTheme } from "@mantine/styles";
```

Comment:
Make sure to respect pascal case for these namings.
Name it like FamilyHistoryIcon, for example.

Reply:
@itsjoeoui I named it "FamilyHistoryIconSvg", is that wrong?

Reply:
@itsjoeoui Are you referring to the file name??